omod/src/main/java/org/openmrs/module/fhir/resources/FHIRConditionResource.java

```diff
@@ -40,6 +40,7 @@ public Condition getByUniqueId(IdType id) {
 	 * @param id id of the OpenMrs Obs
 	 * @return
 	 */
+	@Deprecated
```

Comment:
Deprecating a class and at the same time adding a new method to it?

Reply:
The same class I am deprecating extends the GenericConditionStrategy interface which I am using in condition strategy. Maybe I should delete the class since it wasn't working and It's not useful at all. I have just made it abstract.

Reply:
If we do not need it, then get rid of it.

Reply:
Did you see the above?